maintenance/archives/wikia/patch-populate-rc_ip_bin.sql

```diff
@@ -0,0 +1,4 @@
+-- SUS-3079: Populate rc_ip_bin column
+
+UPDATE /*$wgDBPrefix*/recentchanges
+  SET rc_ip_bin = IFNULL(INET6_ATON(rc_ip), '') WHERE rc_user = 0 AND rc_ip_bin = '';
```

Comment:
Let's make `WikiaUpdater.php` run this SQL as well - maybe in batches of 500 rows?

Reply:
Let's run it later, after we have started inserting data.